teslajsonpy/controller.py

```diff
@@ -390,6 +395,7 @@ async def connect(
         wake_if_asleep: bool = False,
```

Comment:
Is `wake_if_asleep` even used? When looking at the `Controller.api`, `Controller.post` and `Controller.get` methods, I see it's parameter but not used anywhere.

Reply:
It's used by a wakeup decorator. Search and you should see it.

Reply:
Ah got it. I'll have to spend some time understanding how that works!